lib/rspec/rails/view_rendering.rb

```diff
@@ -56,7 +56,7 @@ def self.nullify_template_rendering(templates)
               template.identifier,
               EmptyTemplateHandler,
               :virtual_path => template.virtual_path,
-              :format => template.formats
+              :format => template.respond_to?(:format) ? template.format : template.formats
```

Comment:
Great, is this a Rails specific or object specific? e.g. will we encounter both methods in one version of Rails? If so we prefer conditionally defining a method to extract this to save runtime conditionals.

Reply:
`template.formats` is deprecated in Rails 6, so we could rely on the Rails version instead, perhaps something like this?:

```
def self.template_format(format)
  if Rails::VERSION::STRING >= '6'
    template.format
  else
    template.formats
  end
end
```

Reply:
Right so if this is Rails 6 only we'd typically do:
```
if Rails::VERSION::STRING >= '6'
  def self.template_format(format)
    template.format
  end
else
  def self.template_format(format)
    template.formats
  end
end
```

Reply:
That makes sense, thanks @JonRowe!